src/main/java/core/basesyntax/service/impl/SeparateServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.SeparateService;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class SeparateServiceImpl implements SeparateService {
+    public static final String HEADER = "fruit,quantity";
+
+    @Override
+    public List<String> getReportFromList() {
+        StringBuilder builder = new StringBuilder();
+        builder.append(HEADER)
+                .append(System.lineSeparator());
+        List<String> fromMapToList = new ArrayList<>();
+        for (Map.Entry<String, Integer> output : Storage.fruitStorage.entrySet()) {
```

Comment:
let's create a method getAll in FruitShopDao that returns `Storage.fruitStorage`;